src/view/ChewingEditor.cpp

```diff
@@ -223,8 +230,12 @@ void ChewingEditor::setupImport()
         model_, SIGNAL(importCompleted(bool, const QString&, size_t, size_t)),
         ui_.get()->notification, SLOT(notifyImportCompleted(bool, const QString&, size_t, size_t))
     );
+
+    // initialize import_count
+    import_count = 0;
```

Comment:
The reason why `import_count` should be decoupled out of `src/view/ChewingEditor.h` is simply because of its independency of the interface design of chewing editor.